docs/changelog/1_0_a7.rst

```diff
@@ -0,0 +1,86 @@
+===========
+1.0 Alpha 7
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 7 "Lalande".
+
+
+EdgeQL
+======
+
+* Implement consistent update of expressions when referenced schema
+  objects are renamed (:eql:gh:`#1841`).
+* Add support for per-database configuration by adding a new
+  ``DATABASE`` configuration scope (:eql:gh:`#1867`).
+* Prohibit taking implicit cross products with volatile operations
+  (:eql:gh:`#1784`).
+* Prohibit changing cardinality via inheritance (:eql:gh:`#1772`).
+* Fix moving a type between modules (:eql:gh:`#1890`).
+* Prohibit functions from sharing shortname with types (:eql:gh:`#1465`).
+* Remove legacy unused ``.>`` token (:eql:gh:`#1648`).
+* Disallow partial path expressions in :ref:`LIMIT
+  <ref_eql_statements_select>`/:ref:`OFFSET
+  <ref_eql_statements_select>` clauses (:eql:gh:`#1919`).
+* Update :eql:type:`std::decimal` and :eql:type:`std::bigint` criteria
+  to be sensitive to the presence of ``.`` in the number when
+  determining which of the two arbitrary precision types to use
+  (:eql:gh:`#1804`).
+* Fix some constraint issues (:eql:gh:`#1961`).
+* Prohibit scalar types with more than one concrete base (:eql:gh:`#1790`).
+* Enable DDL for altering enums in order to add new labels (:eql:gh:`#1956`).
+* Support renaming functions (:eql:gh:`#1971`).
+* Fix issues related to changing base types and altering constraints
+  (:eql:gh:`#1996`).
+* Fix functions returning a set of tuples (:eql:gh:`#2010`).
+* Fix cardinality inference on operators (:eql:gh:`#2001`).
+* Implementation of more of the features of the `RFC 1000
+  <migrations_>`_ migration syntax, such as improving the proposed
+  migration DDL, adding migration hash, fixing issues with renames,
+  fixing issues with changing bases, improving expression dependency
+  tracing, improving handling changes to aliases, adding DDL commands
+  to better control fields like ``errmessage`` or ``default`` and
+  other changes (:eql:gh:`#1987`).
+
+
+Command-Line Tools
+==================
+
+* Deprecate ``--default-database`` and ``--default-database-user``
+  (:eql:gh:`#1879`).
+* Add a new header to the ``Prepare`` message to implicitly inline
+  type names so they can be used by REPL (or similar tools)
+  (:eql:gh:`#1567`).
+* Improve accuracy of syntax error reporting (:eql:gh:`#1959`).
+
+
+Bindings
+========
+
+We now have an improved spec for client API (`RFC 1004 <robust_>`_).
+Rolling out the support for the full spec will be done in the next
+release, but some implementation work has already started.
+
+* Initial implementation of capabilities from `RFC 1004 <robust_>`_
```

Comment:
remove this

Reply:
I was thinking that much like we had an ongoing effort to implement RFC1000 mentioned in past changelogs (alpha 4, 5, 6), a similar mention is worth for starting implementaiton of RFC1004.

Reply:
This has no effect on user APIs -- it's an internal change.

Reply:
Removed